examples~/quickstart/client/README.md

```diff
@@ -0,0 +1,13 @@
+# Quickstart client
+See [SpacetimeDB](https://github.com/clockworklabs/SpacetimeDB)/modules/quickstart-chat
+
+## Regenerating bindings
+
+To regenerate bindings: clone SpacetimeDB next to this repo, then in this directory:
+
+```bash
+rm -rf module_bindings/*
```

Comment:
Nit: this shouldn't be necessary, `spacetime generate` automatically deletes obsolete files, just pass `-y` for confirmation.

Reply:
Ah, I'd already written a script for this anyway on another branch, brought that in.